gnovm/cmd/gno/test_test.go

```diff
@@ -90,82 +91,82 @@ func TestTest(t *testing.T) {
 		},
 		{
 			args:                []string{"test", "--verbose", "--run", ".*/NoExists", "../../../examples/gno.land/p/demo/ufmt"},
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                []string{"test", "--verbose", "--run", ".*/hello/NoExists", "../../../examples/gno.land/p/demo/ufmt"},
 			stdoutShouldContain: "RUN   TestSprintf",
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                []string{"test", "--verbose", "--run", "Sprintf/", "../../../examples/gno.land/p/demo/ufmt"},
 			stdoutShouldContain: "RUN   TestSprintf",
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                []string{"test", "--verbose", "--run", "Sprintf/.*", "../../../examples/gno.land/p/demo/ufmt"},
 			stdoutShouldContain: "RUN   TestSprintf",
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                []string{"test", "--verbose", "--run", "Sprintf/hello", "../../../examples/gno.land/p/demo/ufmt"},
 			stdoutShouldContain: "RUN   TestSprintf",
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                []string{"test", "--verbose", "--timeout", "1000000s", "../../../examples/gno.land/p/demo/ufmt"},
 			stdoutShouldContain: "RUN   TestSprintf",
-			stderrShouldContain: "ok      ./../../../examples/gno.land/p/demo/ufmt",
+			stderrShouldContain: "ok      ../../../examples/gno.land/p/demo/ufmt",
 		},
 		{
 			args:                 []string{"test", "--verbose", "../../tests/integ/native-lib"},
-			recoverShouldContain: "./../../tests/integ/native-lib/contract.gno:1: unknown import path net",
+			recoverShouldContain: "../../tests/integ/native-lib/contract.gno:1: unknown import path net",
 		},
 		{
 			args:                []string{"test", "--verbose", "--with-native-fallback", "../../tests/integ/native-lib"},
-			stderrShouldContain: "ok      ./../../tests/integ/native-lib",
+			stderrShouldContain: "ok      ../../tests/integ/native-lib",
 		},
 		{
 			args:                 []string{"test", "--verbose", "../../tests/integ/unknown-lib"},
-			recoverShouldContain: "./../../tests/integ/unknown-lib/contract.gno:1: unknown import path foobarbaz",
+			recoverShouldContain: "../../tests/integ/unknown-lib/contract.gno:1: unknown import path foobarbaz",
 		},
 		{
 			args:                 []string{"test", "--verbose", "--with-native-fallback", "../../tests/integ/unknown-lib"},
```

Comment:
Can you add more tests with the ellipsis pattern (`./foo/...`).

Reply:
And where the pattern is a single file `./path/to/file.gno` or multiple files in the same dir `./path/to/z*.gno`.

Reply:
Marking as resolved as i removed pattern support from this PR.